src/lpython/tests/test_llvm.cpp

```diff
@@ -1553,112 +1553,112 @@ TEST_CASE("PythonCompiler tuples") {
+//     x: i32
+// )");
+//     CHECK(r.ok);
+//     CHECK(r.result.type == PythonCompiler::EvalResult::none);
+
+//     r = e.evaluate2("c1: MyClass1 = MyClass1(12)");
+//     CHECK(r.ok);
+//     CHECK(r.result.type == PythonCompiler::EvalResult::statement);
+
+//     r = e.evaluate2("c1");
+//     CHECK(r.ok);
+//     CHECK(r.result.type == PythonCompiler::EvalResult::struct_type);
+//     CHECK(e.aggregate_type_to_string(r.result) == "MyClass1(x=12)");
+
+//     r = e.evaluate2(R"(
+// @dataclass
+// class MyClass2:
+//     i: i32
+//     f: f64
+// )");
+//     CHECK(r.ok);
+//     CHECK(r.result.type == PythonCompiler::EvalResult::none);
+
+//     r = e.evaluate2("c2: MyClass2 = MyClass2(12, 2.5)");
+//     CHECK(r.ok);
+//     CHECK(r.result.type == PythonCompiler::EvalResult::statement);
+
+//     r = e.evaluate2("c2");
+//     CHECK(r.ok);
+//     CHECK(r.result.type == PythonCompiler::EvalResult::struct_type);
+//     CHECK(e.aggregate_type_to_string(r.result) == "MyClass2(i=12, f=2.500000)");
+
+//     r = e.evaluate2(R"(
+// @dataclass
+// class MyClass3:
+//     i: i32
+//     f: f64
+//     s: str
+// )");
+//     CHECK(r.ok);
+//     CHECK(r.result.type == PythonCompiler::EvalResult::none);
+
+//     r = e.evaluate2("c3: MyClass3 = MyClass3(12, 2.5, \"LPython\")");
+//     CHECK(r.ok);
+//     CHECK(r.result.type == PythonCompiler::EvalResult::statement);
+
+//     r = e.evaluate2("c3");
+//     CHECK(r.ok);
+//     CHECK(r.result.type == PythonCompiler::EvalResult::struct_type);
+//     CHECK(e.aggregate_type_to_string(r.result) == "MyClass3(i=12, f=2.500000, s=\"LPython\")");
+
+//     r = e.evaluate2(R"(
+// @dataclass
+// class MyClass4:
+//     i_1: bool
+//     i_8: i8
+//     i_16: i16
+//     i_32: i32
+//     i_64: i64
+// )");
+//     CHECK(r.ok);
+//     CHECK(r.result.type == PythonCompiler::EvalResult::none);
+
+//     r = e.evaluate2("c4: MyClass4 = MyClass4(True, i8(2), i16(3), i32(4), i64(5))");
+//     CHECK(r.ok);
+//     CHECK(r.result.type == PythonCompiler::EvalResult::statement);
+
+//     r = e.evaluate2("c4");
+//     CHECK(r.ok);
+//     CHECK(r.result.type == PythonCompiler::EvalResult::struct_type);
+//     // CHECK(e.aggregate_type_to_string(r.result) == "MyClass4(i_1=True, i_8=2, i_16=3, i_32=4, i_64=5)"); // FIXME: look at issue #2793
+
+//     r = e.evaluate2(R"(
+// @dataclass
+// class MyClass5:
+//     u_1: bool
+//     u_8: u8
+//     u_16: u16
+//     u_32: u32
+//     u_64: u64
+// )");
+//     CHECK(r.ok);
+//     CHECK(r.result.type == PythonCompiler::EvalResult::none);
+
+//     r = e.evaluate2("c5: MyClass5 = MyClass5(False, u8(2), u16(3), u32(4), u64(5))");
+//     CHECK(r.ok);
+//     CHECK(r.result.type == PythonCompiler::EvalResult::statement);
+
+//     r = e.evaluate2("c5");
+//     CHECK(r.ok);
+//     CHECK(r.result.type == PythonCompiler::EvalResult::struct_type);
+//     CHECK(e.aggregate_type_to_string(r.result) == "MyClass5(u_1=False, u_8=2, u_16=3, u_32=4, u_64=5)");
+// }
```

Comment:
This single ctest is commented out. I believe it is flaky (sometimes passes, sometimes fails).